### PR TITLE
Truncate long relay URLs

### PR DESCRIPTION
**Pull request by erskingardner**
This fixes long URLs overrunning

image

image

### CODE REVIEW

**Comment by erskingardner** [comment]
Also added a simple check to say "subscription" or "subscriptions" based on the count.

File: src/lib/utils/relay/index.ts [hunk 1]
```diff
@@ -16,5 +16,10 @@ export function formatRelayName(relay: NDKRelay): string {
     // strip protocol prefix
     name = name.replace(/^(ws|wss):\/\//, '');
 
+    // truncate long urls
+    // e.g. filter.nostr.wine/{npub}?broadcast=true
+    if (name.length > 28) {
```
Comment (pablof7z) [inline src/lib/utils/relay/index.ts:21]: probably better to do it with CSS?

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Comment (erskingardner) [reply src/lib/utils/relay/index.ts:21]: I was trying to leave the styling as minimal as possible. Maybe we just shouldn't truncate at all and let the use pass in styling themselves?

Comment (erskingardner) [reply src/lib/utils/relay/index.ts:21]: Ah, an issue with that is that the user won't have an easy way to target the HTML element that houses the relay URL specifically.

**Comment by erskingardner** [comment]
@pablof7z Finally came back to this PR and sorted the truncation.

**Comment by pablof7z** [comment]
squash merged due to conflicts

**Pull request closed without merging**